chainguard-keys.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: chainguard-keys
+  version: 1
+  epoch: 5
+  description: "Chainguard public keys"
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - busybox
+
+pipeline:
+  - name: Install
+    runs: |
+      mkdir -p "${{targets.destdir}}"/etc/apk/keys
+      install -m644 chainguard-extras.rsa.pub "${{targets.destdir}}"/etc/apk/keys
+      install -m644 chainguard-enterprise.rsa.pub "${{targets.destdir}}"/etc/apk/keys
+
+      for i in x86_64 aarch64; do
+        mkdir -p "${{targets.destdir}}"/usr/share/apk/keys/${i}
+        install -m644 chainguard-extras.rsa.pub "${{targets.destdir}}"/usr/share/apk/keys/${i}/chainguard-extras.rsa.pub
+        install -m644 chainguard-enterprise.rsa.pub "${{targets.destdir}}"/usr/share/apk/keys/${i}/chainguard-enterprise.rsa.pub
+      done
```

Comment:
I believe apko also sets up `/etc/apk/keys`, so I would be curious which one "wins" in these cases.  cc @jonjohnsonjr

Reply:
Ack this is based on `wolfi-keys`, which would already have this problem (if it's a problems).

Reply:
Yeah, I literally cloned wolfi-keys.yaml, and assumed that it does the right thing...

Reply:
I _think_ this should be fine but I can dig in if something goes wonky.